NastassiaBon/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def data_extraction
+  file = File.new('data.txt')
+  file.read
+end
+
+def mean(town, data)
+  return -1.0 unless data.include? town
+
+  serch_town(town, data).sum / 12
+end
+
+def variance(town, data)
+  return -1.0 unless data.include? town
+
+  serch_town(town, data).map { |i| (mean(town, data) - i)**2 }.sum / 12
+end
+
+def serch_town(town, data)
+  data.split("\n").each do |i|
```

Comment:
Better not to use one-letter variables. What is `i` in your data? City? So rename it into the `city`